app/resources/add_note.html.template

```diff
@@ -8,281 +8,76 @@
+        if (typeof(id_or_elem) == 'string') {
+          document.getElementById(id_or_elem).style.display = hide_text;
+        } else {
+          id_or_elem.style.display = hide_text;
+        }
+      }
+
+      function hide_unhide_note_contents(note_contents_id) {
+        var note = document.getElementById(note_contents_id + '-contents');
+        var hidden = note.style.display == 'none';
+        set_display(note, !hidden);
+        set_display('reveal-note', hidden);
+        set_display('hide-note', !hidden);
+        set_display(note_contents_id + '-mark-not-spam', !hidden);
+      }
+    </script>
   {% endif %}
-  <div class="mandatory text section">
-    <h3>
-      {% trans "Message (required)" %}
-    </h3>
-    <div class="hint">
-      {% if not markdup %}
-        {% trans "A message for this person or others seeking this person" %}
-      {% else %}
-        {% trans "Please explain why you think these are the same person" %}
-      {% endif %}
-    </div>
-    <div class="field">
-        <textarea name="text" id="text"
-          class="long-text-input"
-          rows=5>{{params.text}}</textarea>
-    </div>
-    <div class="end-multi-columns"></div>
-  </div>
-  {% if not markdup %}
-    <div class="location section">
-      <h3>
-        {% trans "Last known location" %}
-      </h3>
-      <div class="hint">
-        {% if env.enable_javascript and env.maps_api_key %}
-          {% trans "Type an address or open the map below and indicate the location by clicking on the map." %}
-        {% else %}
-          {% trans "Type an address." %}
-        {% endif %}
-      </div>
-      <div class="field">
-          <textarea name="last_known_location"
-              id="clickable_map_location_field"
-              class="long-text-input"
-              rows=2>{{params.last_known_location}}</textarea>
-      </div>
-      <div class="end-multi-columns"></div>
-      {% if env.enable_javascript and env.maps_api_key %}
-        <div class="field map-container">
-          <div id="clickable_map_show_link">
-            <a href='#' onclick="toggleClickableMap('clickable_map'); return false;">
-              {% trans "Show Map" %}</a>
+{% endblock head %}
+
+{% block content %}
+  {% if not standalone %}
+    {% include "map.html.template" %}
+  {% endif %}
+  <form method="post" enctype="multipart/form-data" action="{{env.repo_path}}/view">
+    {{env.hidden_input_tags_for_preserved_query_params|safe}}
+    <input type="hidden" name="id" value="{{person.record_id}}">
+    <input type="hidden" name="query" value="{{params.query}}">
+    <div class="view">
+
+      <div class="column centered">
+        {% if person.notes_disabled %}
+          <div class="mandatory">
+            {% trans "The author has disabled notes on this record." %}
+            <p>
+            <input type="button"
+              value="{% trans "Enable notes on this record" %}"
+              onclick="javascript:location.href='{{enable_notes_url}}'"
```

Comment:
Would this button work for old tier2 phones that do not support javascript? Since there are many of those phones still in emerging markets, where Person Finder is used, we should gracefully fallback when javascript is not supported at least for core user flows.